doc/release-notes.md

```diff
@@ -14,335 +14,215 @@ To receive notifications about updates, subscribe to the release mailing list:
+
 
-Dogecoin Core should also work on most other Unix-like systems but is not
-frequently tested on them.
 
 Notable changes
-===============
+================
 
 Important Security Updates
 --------------------------
 
-This release contains fixes that harden node and network security. These fixes
-are important for every node operator and wallet user.
-
-### Alert system removal
-
-The P2P alert system has been removed and alert messages are no longer
-processed. This was planned for the next major release, but as bugs were found
-in the implementation, the full deprecation of this feature has been executed
-early, because it makes no sense to overhaul a system that is unused and
-already marked for removal in the near future.
-
-### Harden the transaction download mechanism
-
-Significantly reduce the impact of peers that withhold transaction information
-(accidentally or otherwise) in the announcement stage, before the transaction
-is mined, by enforcing strict controls, limits and timeouts on all transaction
-announcements and giving preference to outgoing connections when deciding which
-peer to request transaction information from.
-
-This improves the resilience of the entire network and improves reliability of
-transaction relay throughout.
-
-### Other security-related fixes
-
-- Harden the protocol implementation to reject buggy or malformed messages.
-  This helps protect the network against broken custom clients.
-- Limit and tightly manage memory usage in events of high network traffic or
-  when connected to extremely slow peers. This protects nodes on lower end
-  hardware to not run out of memory in the face of increased network activity.
-
-Fee Recommendation
-------------------
-
-This release changes the recommended dust limit for all participants on the
-Dogecoin network from 1 DOGE to 0.01 DOGE. The full recommendation can be found
-[in the documentation](fee-recommendation.md).
-
-This change has been implemented in the wallet as the default value of
-`-discardthreshold`, resulting in the following default logic:
-
-1. The wallet will attempt to not create any outputs smaller than 0.03 DOGE as
-   change if possible.
-2. If after signing there is more than 0.01 DOGE left in the change output, the
-   wallet will keep the output as-is.
-3. If the change output drops under 0.01 DOGE, the wallet will discard the
-   change and instead give it to miners as fee.
-4. Any output under 0.01 DOGE will not be accepted as valid by default.
-
-Breaking changes
-----------------
-
-### Configuration of a wallet backup directory
-
-Adds a new command line parameter `-backupdir` that allows operators to specify
-where to store wallet backups and dumps. When not set, the node will add a
-directory under the data directory called "backups".
-
-**This is a breaking change.**
+This release contains fixes for Dogecoin Qt across all platforms that increase
+security for end users.
```

Comment:
#3352 The RPC authentication method has been updated and hardened to use cryptographically secure random numbers for generating passwords.

Reply:
nvmd this is mentioned on line 209

Reply:
I can elaborate on that a bit more in the item there. But let me sleep on formulation.

Reply:
@patricklodder I don't know how you manage to bat 1.000 with all you do here but you do. Bravo brother!

Reply:
Bumped the item to "security fixes" and explained how it can be used.